src/models/CampaignSharing.ts

```diff
@@ -31,15 +31,15 @@ interface UserGroupsSharingFilter {
 interface UsersByOrgUnitsSharingFilter {
     type: "usersByOrgUnits";
     level: number;
-    userRoles: string[];
+    userRolesGroups: Array<string[]>;
```

Comment:
@tokland userRolesGroups? A bit strange, isn't it? Aren't they userRoles?

Reply:
Indeed, that name makes no sense whatsoever, fixed: 89b3ef64